app/src/main/java/com/mobiledevelopmentfoodapp/dto/Customer.kt

```diff
@@ -4,7 +4,12 @@ package com.mobiledevelopmentfoodapp.dto
 //import androidx.room.PrimaryKey
 import com.google.gson.annotations.SerializedName
 
-data class Customer(var name: String, var phone: String, @SerializedName("id") var customerId:Int = 0 ) {
+data class Customer(
```

Comment:
re-organized data class to seperate it into multiple lines, also added trailing comma

Reply:
Duplicates a previous review.  When that happens, first in wins.